test/245_dns_case_insensitive_test.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+. ./config.sh
+
+C1=10.2.0.78
+C2=10.2.0.79
+C3=10.2.0.80
+
+start_suite "DNS lookup case (in)sensitivity"
+
+weave_on $HOST1 launch
+
+start_container_with_dns $HOST1        --name=test
+
+start_container $HOST1 $C1/24 --name=seeone
+assert_dns_record $HOST1 test seeone.weave.local $C1
+assert_dns_record $HOST1 test SeeOne.weave.local $C1
+assert_dns_record $HOST1 test SEEONE.weave.local $C1
```

Comment:
We need 3 dns entries for this test to properly test the sort order; this provides the dns entry that would split SeEtWo and seetwo if sorting isn't working.

I guess we can remove the ```assert_dns_record ...  seetwo.weave.local $C2 $C3``` ones below?  But I like the completeness.